static/app/views/settings/project/server-side-sampling/modals/uniformRateChart.tsx

```diff
@@ -25,7 +24,7 @@ function UniformRateChart({series, isLoading}: Props) {
   return (
     <ChartPanel>
       <ChartContainer>
-        <TransitionChart loading={isLoading} reloading={false} height="224px">
+        <TransitionChart loading={false} reloading={false} height="224px">
```

Comment:
Would there be a visual difference if we removed this completely?

Reply:
yes. Since we are displaying the loading indicator before rendering the chart, it seems to be fine without it. Thanks for the feedback